config

```diff
@@ -1,4 +1,4 @@
 baseuri     https://nodejs.org/dist
-default_variant stretch
+default_variant buster
```

Comment:
I'm fine with adding a buster variant, but please don't change the default version.